src/supervisor.tex

```diff
@@ -1100,6 +1114,11 @@ \subsection{Supervisor Memory-Management Fence Instruction}
 In this case, a similar remark applies: it is unpredictable whether the old
 non-leaf PTE or the new leaf PTE is used, but the behavior is otherwise well
 defined.
+
+Another consequence of this specification is that it is generally unsafe to
+update a PTE using a set of stores of a width less than the width of the PTE,
+as it is legal for the implementation to read the PTE at any time, including
+when only some of the partial stores have taken effect.
```

Comment:
Do we state somewhere that the atomicity of an implicit PTE read (or write in the case of the A/D bits) is the same as the size of the PTE? Or are we not requiring that?